Source/Core/Core/HotkeyManager.cpp

```diff
@@ -37,7 +37,10 @@ constexpr std::array<const char*, NUM_HOTKEYS> s_hotkey_labels{{
     _trans("Unlock Cursor"),
     _trans("Center Mouse"),
     _trans("Activate NetPlay Chat"),
-    _trans("Control NetPlay Golf Mode"),
+    _trans("Control NetPlay Golf Mode"),  
+#ifdef USE_RETRO_ACHIEVEMENTS
+    _trans("Open Achievements"),
+    #endif  // USE_RETRO_ACHIEVEMENTS
```

Comment:
Something... weird is going on with the formatting here. Why is the first line marked as different, and why is the indent on the `#endif` wrong? Lint isn't complaining about this either which confuses me.

Reply:
The first line had hidden space at the end of the line. Why the linter never picked up on it is beyond me.